### ISSUE TITLE
(Found resolution) Images to South-West & Rectangle Points to North-East [r52]

### ISSUE THREAD

**Issue by XpMonster** [Scirra/Construct-3-bugs#764]
## Problem description

It only affects the editor in version r52. All fine in the game. 

Images moving to the left-bottom side and their rectangle points are moving to the right-top side of the object in case if any effect applied on any layer (Or simply if any layer has it's own texture)

That bug also doesn't cares on which layer objects are. So it seen on objects that are on layer above and below the layer with effect applied.

Also if you remove causers it won't return to normal until you reload the project.

And also It maybe has some influence on performance. 
I had less CPU used before r52 (Worked on 51r few minutes before updating)

***UPDATE*** I decided to check how it works with Themes and it appeared that there's no such bug in Dark Theme. Then I moved to Light theme. No bug. Then finally back to default and ...no bug! So seems like there was something stored in Construct's cash that was supposed to change, but didn't. 

So if anyone else has this bug here's a solution - install another theme, reload C3, pick after any another theme, reload C3 and all will work. 

## Attach a .c3p

[Check it here](https://drive.google.com/open?id=0Bz-IOXMAnpNvM3JhUE9kenZnMFU)

## Steps to reproduce

1. Apply effect on any layer 
2. Pick any object
3. Find rectangle points 

## How to Fix

1. Pick another Theme in Settings and reload Construct 3
2. After that you can work in any theme you want to. Even in the one you had this bug. 

## Observed result

<img src="http://i.imgur.com/FkU3ptB.png" alt="Observed bug" style="width:280px;height:280px;">

## Expected result

Have it back to normal.

## Affected browsers

* Chrome: yes
* Firefox: yes
* Edge: yes
* Safari: yes

**Comment by AshleyScirra** [comment]
It looks fine here unless I resize the window smaller. If it only happens after resizing the window it's probably a duplicate of #671.

**Comment by XpMonster** [comment]
I didn't changed window resolution before, but I tried that now when you said it. It's not the same.

**Comment by AshleyScirra** [comment]
It can happen any time the layout view changes size, e.g. by resizing side panes, going fullscreen and back again, etc. Additionally reloading C3 resets the size so it will work again, which is probably why you thought changing theme fixes it (since you have to reload C3 to change theme). I am pretty sure it's related to bug #671 so closing as a duplicate.

**Comment by XpMonster** [comment]
You're wrong here. I have reloaded C3 few times and it had no effect. 
I tried themes just for fun to see them and was super surprised it fixed my problem.

**Comment by AshleyScirra** [comment]
I can't reproduce any problem with your example project, unless I first resize the window smaller, at which point it looks exactly like what you report. So it appears to me to be identical to issue #671. I'll focus on fixing that first, but if that does not also solve this problem then I'll take another look.

**Comment by XpMonster** [comment]
Resizing the screen does somewhat close, but quite different to what I described and 10 times uglier ;/
My bug worked precisely with layers, if I added more layers with their own textures, effects - it moved both rectangle points and image away further from the object and only in sides pointed out in description. 

Anyway I'm very glad I found this strange resolution. If anyone appears to have the same problem I suggest you advice him to do the same. 

It one-time thing as I can't reproduce it anymore.
